packages/bpk-component-spinner/src/BpkSpinner.js

```diff
@@ -26,18 +28,24 @@ import SPINNER_TYPES from './spinnerTypes';
 
 const getClassName = cssModules(STYLES);
 
-const BpkSpinner = props => {
+type Props = {
+  type: string,
```

Comment:
Same comment as earlier about using `$Keys<typeof SPINNER_TYPES>`.

Reply:
Did this suggestion and seems to work fine